packages/react-devtools-shared/src/devtools/views/Components/reach-ui/menu-button.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
```

Comment:
Note: I have added this by just copying pasting it from other files. Let me know if any change is required here.